aws/resource_aws_vpc_test.go

```diff
@@ -573,6 +573,10 @@ func TestAccAWSVpc_classiclinkDnsSupportOptionSet(t *testing.T) {
 const testAccVpcConfig = `
 resource "aws_vpc" "test" {
   cidr_block = "10.1.0.0/16"
+
+  tags = {
+    Name = "terraform-testacc-vpc"
+  }
```

Comment:
This addition is extraneous for a basic test and seems to have caused this new test failure:

```
=== CONT  TestAccAWSVpc_basic
TestAccAWSVpc_basic: resource_aws_vpc_test.go:108: Check 4/13 error: aws_vpc.test: Attribute 'tags.%' expected "0", got "1"
--- FAIL: TestAccAWSVpc_basic (17.45s)
```

```suggestion
```

Reply:
```
--- PASS: TestAccAWSVpc_basic (29.15s)
```